py/src/pathfinder_worker/call.py

```diff
@@ -516,22 +542,79 @@ def loop_inner(
 
 
 def render(verb, vals):
```

Comment:
Instead of hand-coding serialization here couldn't we just use marshmallow to serialize the `FeeEstimationInfo` and `TransactionSimulationInfo` objects we already have? That might mean that we have to make some changes to the types we use for serializing responses on the Rust side, but we seem to have those types anyway, so maybe it's just simpler to have this in one place?

Reply:
I'm going to follow up on this: https://github.com/eqlabs/pathfinder/issues/987

Reply:
Thanks. I did not do this when converting the input side from manual to marshmallow... :(

Reply:
Seems like we're stuck with manual rendering: https://github.com/eqlabs/pathfinder/issues/987#issuecomment-1487308236

Reply:
Solved in https://github.com/eqlabs/pathfinder/pull/977/commits/67fb7cea8df2b1293a5aef811904e432a39563ee